Dalitz

remitche66 opened the pull request:
Added a python script to create a new package based on the Dalitz tutorial and made minor updates to the Dalitz tutorial documentation.

mashephe commented:
Ryan,

Does your script change class names inside of the file also?

For example, DalitzPlotGenerator -> Psi3BodyPlotGenerator?

And how about library names embedded in Makefiles?

Matt


On Apr 9, 2020, at 12:38 PM, Ryan Mitchell <notifications@github.com<mailto:notifications@github.com>> wrote:


Added a python script to create a new package based on the Dalitz tutorial and made minor updates to the Dalitz tutorial documentation.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/mashephe/AmpTools/pull/21

Commit Summary

  *   Added python script to make a new package based on the Dalitz tutorial.
  *   Minor updates to the documentation.

File Changes

  *   A Tutorials/Dalitz/copyDalitz.py<https://github.com/mashephe/AmpTools/pull/21/files#diff-90df2aeb0bac7063cc2b376794449098> (65)
  *   M Tutorials/Dalitz/doc/doc.pdf<https://github.com/mashephe/AmpTools/pull/21/files#diff-4badeef988779b415b237803b3bfd28f> (0)
  *   M Tutorials/Dalitz/doc/doc.tex<https://github.com/mashephe/AmpTools/pull/21/files#diff-623615d6ae05a8ab3adf73878636d044> (6)

Patch Links:

  *   https://github.com/mashephe/AmpTools/pull/21.patch
  *   https://github.com/mashephe/AmpTools/pull/21.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mashephe/AmpTools/pull/21>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADG5L2AD25IQ5QBNHNZGEZ3RLX2YHANCNFSM4ME3WA6A>.

remitche66 commented:
Hi Matt,

Yes, it changes all "Dalitz" to "Psi3Body", all "dalitz" to "psi3body", and all "DALITZ" to "PSI3BODY" for the file names and also everywhere inside all the files, including Makefiles. The only very minor hiccup I found is that one of the histogram titles becomes "Psi3Body plot", instead of "Dalitz plot", even though it's really still a Dalitz plot.

Another caveat, the script will try to do the same thing for binary files, but will crash -- so a "make clean" is really needed before running the script.    There's apparently a way to check for binaries in python3, but apparently not in python2, and python2 seems safer (it's the default on a mac).

Ryan


On Apr 9, 2020, at 5:24 PM, Matthew Shepherd <notifications@github.com<mailto:notifications@github.com>> wrote:



Ryan,

Does your script change class names inside of the file also?

For example, DalitzPlotGenerator -> Psi3BodyPlotGenerator?

And how about library names embedded in Makefiles?

Matt


On Apr 9, 2020, at 12:38 PM, Ryan Mitchell <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


Added a python script to create a new package based on the Dalitz tutorial and made minor updates to the Dalitz tutorial documentation.

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/mashephe/AmpTools/pull/21

Commit Summary

* Added python script to make a new package based on the Dalitz tutorial.
* Minor updates to the documentation.

File Changes

* A Tutorials/Dalitz/copyDalitz.py<https://github.com/mashephe/AmpTools/pull/21/files#diff-90df2aeb0bac7063cc2b376794449098> (65)
* M Tutorials/Dalitz/doc/doc.pdf<https://github.com/mashephe/AmpTools/pull/21/files#diff-4badeef988779b415b237803b3bfd28f> (0)
* M Tutorials/Dalitz/doc/doc.tex<https://github.com/mashephe/AmpTools/pull/21/files#diff-623615d6ae05a8ab3adf73878636d044> (6)

Patch Links:

* https://github.com/mashephe/AmpTools/pull/21.patch
* https://github.com/mashephe/AmpTools/pull/21.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mashephe/AmpTools/pull/21>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADG5L2AD25IQ5QBNHNZGEZ3RLX2YHANCNFSM4ME3WA6A>.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/mashephe/AmpTools/pull/21#issuecomment-611759732>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AIP4MYB2RQ7XBP4J4SLO24TRLY4JRANCNFSM4ME3WA6A>.
